Makefile

```diff
@@ -75,6 +75,11 @@ HELM_MANIFEST_CACHE_CONTROL ?=public, max-age=600
 CONTROLLER_GEN ?=$(GO) run ./vendor/sigs.k8s.io/controller-tools/cmd/controller-gen --
 CRD_PATH ?= pkg/api/scylla/v1/scylla.scylladb.com_scyllaclusters.yaml
 
+SWAGGER ?=./vendor/github.com/go-swagger/go-swagger/cmd/swagger
```

Comment:
swagger command is executed as `$(GO) run ./vendor/github.com/go-swagger/go-swagger/cmd/swagger` (this is a swagger cmd dir)

Reply:
```suggestion
SWAGGER_CMD_PKG ?=./vendor/github.com/go-swagger/go-swagger/cmd/swagger
```